x-pack/plugin/core/src/main/resources/monitoring-apm-server.json

```diff
@@ -0,0 +1,686 @@
+{
+  "index_patterns": [
+    ".monitoring-apm-server-${monitoring.template.version}-*"
+  ],
+  "settings": {
+    "index.auto_expand_replicas": "0-1",
+    "index.codec": "best_compression",
+    "index.format": 6,
+    "index.number_of_replicas": 0,
+    "index.number_of_shards": 1
+  },
+  "version": 7000001,
```

Comment:
FYI: When this is backported, it needs to share the same version as the other templates.